_code-samples/modular-tutorials/account-support.js

```diff
@@ -0,0 +1,158 @@
+// ******************************************************
+// ************* Get the Preferred Network **************
+// ******************************************************  
+
+function getNet() {
+  let net
+  if (document.getElementById("tn").checked) net = "wss://s.altnet.rippletest.net:51233/"
+  if (document.getElementById("dn").checked) net = "wss://s.devnet.rippletest.net:51233/"
+  const client = new xrpl.Client(net)
+  return net
+} // End of getNet()
+
+// *******************************************************
+// ************* Get Account *****************************
+// *******************************************************
+
+async function getAccount() {
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  await client.connect()
+  let results = `\nConnected to ${net}.`
```

Comment:
```suggestion
  let results += `===Getting Account===\nConnected to ${net}.`
```

This suggestion applies to all instances that add to the result field. I think it's worth keeping a log of all actions that take place, so ensuring all results are `+=` and not overridden with `=`. It'd also like to see a more pronounced headers to identify the type of function that's being called to help differentiate info in the logs. So for an offer create transaction, it starts with something like `===Creating Offer===`  and then has a line break for subsequent metadata.

Reply:
I made these changes.